src/test/java/hw9/ValidateUtilTest.java

```diff
@@ -0,0 +1,40 @@
+package hw9;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ValidateUtilTest {
+
+
+    @Test
+    public void isValidPhoneNumber() {
+        assertEquals(true, ValidateUtil.isValidPhoneNumber("+380974929758"));
+        assertEquals(false, ValidateUtil.isValidPhoneNumber("+38097492975h"));
```

Comment:
assertFalse would help you. The same about other tests methods.

Reply:
fixed